frontend/Dockerfile

```
@@ -0,0 +1,18 @@
+# pull official base image
+FROM node:16.13.2-alpine
+
+# set working directory
+WORKDIR /usr/app
+
+# install and cache app dependencies
+COPY package.json .
+COPY package-lock.json .
+RUN npm ci
```

```suggestion
RUN npm ci && npm cache clean
```

This cleans out the cache after an install, which reduces image bloat. We do something similar [here](https://github.com/matrix-org/matrix-hookshot/blob/main/Dockerfile#L33)